zap/src/main/java/org/parosproxy/paros/core/scanner/Variant.java

```diff
@@ -57,6 +58,13 @@ public String setEscapedParameter(
      */
     default void setParameters(HttpMessage message, List<InputVector> inputVectors) {}
 
+    /**
+     * Gets a short Name of the Variant
+     *
+     * @return a {@code String} the short name of the variant
+     */
+    String getShorName();
```

Comment:
- Should be default to not break when called for custom variants added by add-ons.
 - There's a typo in short.

Reply:
The typo seems to impact all variants, since they're overriding.